### PR TITLE
Translation madara beast article.md

### PR DESCRIPTION
**Pull request by joaopereira12**
Please add the labels corresponding to the type of changes your PR introduces:

- Documentation content changes

Resolves: #921 

## What is the new behavior?

madara beast article.md is translated to portuguese.

## Does this introduce a breaking change?

No

### CODE REVIEW

**Comment by d-roak** [comment]
I'll review it

File: docs/content/articles/pt/madara-beast-article.md [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+
+### Inovação
+
+A personalização de appchains permite aos desenvolvedores trabalhar com recursos
+que estão indisponíveis ou arriscados atualmente em ambientes como o Starknet.
+Appchains oferecem a cada equipa a autonomia para escrever e autorizar quaisquer
+dicas de código desejadas. Isto permite appchains desbloquearem muitos casos de
+uso, como ser capaz de reforçar on-chain KYC sem vazar informação privada.
+
+## Efeito do Madara na Pilha da Appchain
```
Comment (d-roak) [inline docs/content/articles/pt/madara-beast-article.md:93]: no Stack para Appchains

File: docs/content/articles/pt/madara-beast-article.md [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+
+1. **Execução:** A camada de execução define a execução dos blocos e geração de
+   diferença de estado. Madara oferece a flexibilidade para mudar entre dois
```
Comment (d-roak) [inline docs/content/articles/pt/madara-beast-article.md:96]: O Madara

File: docs/content/articles/pt/madara-beast-article.md [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+   caixotes de execução,
```
Comment (d-roak) [inline docs/content/articles/pt/madara-beast-article.md:97]: crates

File: docs/content/articles/pt/madara-beast-article.md [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+   [Blockifier por StarkWare](https://github.com/starkware-libs/blockifier) e
```
Comment (d-roak) [inline docs/content/articles/pt/madara-beast-article.md:98]: da

File: docs/content/articles/pt/madara-beast-article.md [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+   [Starknet_in_rust por LambdaClass](https://github.com/lambdaclass/starknet_in_rust).
```
Comment (d-roak) [inline docs/content/articles/pt/madara-beast-article.md:99]: da

File: docs/content/articles/pt/madara-beast-article.md [hunk 6]
```diff
@@ -0,0 +1,157 @@
...
+   Independente da crate escolhida, a estrutura subjacente utiliza a Cairo VM. A
```
Comment (d-roak) [inline docs/content/articles/pt/madara-beast-article.md:100]: do crate escolhido

File: docs/content/articles/pt/madara-beast-article.md [hunk 7]
```diff
@@ -0,0 +1,157 @@
...
+   linguagem Cairo facilita a criação de programas prováveis, permitindo a
+   demonstração da execução correta do cálculo.
+2. **Settlement:** Como um settlement rollup, o estado da appchain do Madara
```
Comment (d-roak) [inline docs/content/articles/pt/madara-beast-article.md:103]: validity rollup

File: docs/content/articles/pt/madara-beast-article.md [hunk 8]
```diff
@@ -0,0 +1,157 @@
...
+   pode ser reconstruído exclusivamente ao examinar a sua settlement layer. Ao
+   se estabelecer mais frequentemente na Starknet L2, uma appchain L3 consegue
+   alcançar uma finalidade dura mais rápido, enquanto a descentralização da
+   camada de sequenciamento permite uma finalidade mais robusta e suave. Assim,
+   a settlement é reforçada em ambas as frentes (finalidade dura e suave).
+3. **Sequenciamento:** O Madara assume o controlo do processo de sequenciação,
```
Comment (d-roak) [inline docs/content/articles/pt/madara-beast-article.md:109]: u need to be consistent, pick either "sequencer" or "sequenciamento" through the whole document. Probably the pt alternative is better

File: docs/content/articles/pt/madara-beast-article.md [hunk 9]
```diff
@@ -0,0 +1,157 @@
...
+   que pode ser alterado em função das necessidades da aplicação - seja o
+   simples FCFS, PGA ou esquemas mais complexos como Narwhall & Bullshark.
+   Certas appchains podem escolher implementar mempools encriptados para
```
Comment (d-roak) [inline docs/content/articles/pt/madara-beast-article.md:112]: encriptadas

File: docs/content/articles/pt/madara-beast-article.md [hunk 10]
```diff
@@ -0,0 +1,157 @@
...
+   garantir uma ordenação justa e mitigar o impacto do MEV.
+4. **Disponibilidade de Dados:** A Disponibilidade de dados garante o acesso à
+   árvore de estados completa, oferecendo aos utilizadores a confiança de que
+   podem provar a posse dos seus fundos mesmo se o Madara sofrer uma
+   interrupção. Madara irá oferecer aos desenvolvedores uma data de esquemas de
+   disponibilidade de dados para escolherem.
+5. **Controlo:** Cada appchain do Madara pode escolher o seu modelo de controlo.
```
Comment (d-roak) [inline docs/content/articles/pt/madara-beast-article.md:119]: Governança != Controlo, in all paragraph

File: docs/content/articles/pt/madara-beast-article.md [hunk 11]
```diff
@@ -0,0 +1,157 @@
...
+   controlo alternativos também estão sob exploração, como o native substrate
+   governance pallet. Controlo on-chain é um valor fundamental para o Madara.
+
+![come come](https://lh4.googleusercontent.com/i7bXi2IPV-LTLzEgueA2SPHGULUFDj1OX4IznOQr5BeZe0hcey-VXA5TOV6q9XaVqBGAcYiie7u7uxw7q1ByZxjkPQKHERqKJTxhdDdTSgBQy8smyNO3jEHiNJv7Eqh8BMxjj4fFlQAW6gm-hQMzyIU)
+
+## Enter: Madara
+
+No Madara, a Cairo VM está a ser aprimorada utilizando a framework Substrate e
+integrando a Cairo VM para executar programas Cairo e smart contracts da
+Starknet. Substrate é uma framework do Rust open-source para construir
```
Comment (d-roak) [inline docs/content/articles/pt/madara-beast-article.md:131]: O Substrate ... em Rust

File: docs/content/articles/pt/madara-beast-article.md [hunk 12]
```diff
@@ -0,0 +1,157 @@
...
+blockchains personalizáveis, que é conhecida pela sua flexibilidade. Enquanto
+isso, a Cairo VM é desenhada especificamente para gerar Provas de Validade
+eficientemente para execução de programas. Ao implementar rastreamento de estado
```
Comment (d-roak) [inline docs/content/articles/pt/madara-beast-article.md:134]: a execução

File: docs/content/articles/pt/madara-beast-article.md [hunk 13]
```diff
@@ -0,0 +1,157 @@
...
+e um smart contract para verificar essas provas na L2, a appchain garante uma
+integração segura com a Starknet. Desta maneira, o Madara aproveita o poder do
+Cairo para permitir a comprovação da execução do programa.
+
+A natureza modular inerente da framework Substrate permite ao desenvolvedores
+personalizar a appchain com facilidade. Nenhuma suposição é imposta, permitindo
+que você incorpore seus protocolo de consenso próprio, funções hash, esquema de
```
Comment (d-roak) [inline docs/content/articles/pt/madara-beast-article.md:141]: permitindo a incorporação dos teus protocolos de consenso, funções ... 
u started informal and finished formal?

File: docs/content/articles/pt/madara-beast-article.md [hunk 14]
```diff
@@ -0,0 +1,157 @@
...
+assinatura, layout de armazenamento – o que quer que seu aplicativo exija,
```
Comment (d-roak) [inline docs/content/articles/pt/madara-beast-article.md:142]: armazenamento - independente das necessidades da aplicação, utilizando o Cairo para gerar provas

File: docs/content/articles/pt/madara-beast-article.md [hunk 15]
```diff
@@ -0,0 +1,157 @@
...
+enquanto utiliza o Cairo para gerar provas. Sem limites no que os
+desenvolvedores podem fazer enquanto ainda são demonstráveis, herdando o
```
Comment (d-roak) [inline docs/content/articles/pt/madara-beast-article.md:144]: fazer, equanto ainda são comprováveis, herdando a segurança da chain ...

File: docs/content/articles/pt/madara-beast-article.md [hunk 16]
```diff
@@ -0,0 +1,157 @@
...
+segurança da cadeia subjacente - seja Starknet ou Ethereum.
+
+Inicialmente, Madara terá uma forte semelhança com o Starknet, permitindo a
```
Comment (d-roak) [inline docs/content/articles/pt/madara-beast-article.md:147]: o Madara ... a Starknet

File: docs/content/articles/pt/madara-beast-article.md [hunk 17]
```diff
@@ -0,0 +1,157 @@
...
+capacidade de composição de smart contracts no ecossistema da Starknet.
+Encontram-se guardados planos maiores para o futuro à medida que a Starknet se
```
Comment (d-roak) [inline docs/content/articles/pt/madara-beast-article.md:149]: Existem planos...

**Review by d-roak — CHANGES_REQUESTED** [review]

**Review by d-roak — CHANGES_REQUESTED** [review]
double check consistency along the translation

**Comment by tdelabro** [comment]
@d-roak is this ready to be merged?

**Comment by d-roak** [comment]
> @d-roak is this ready to be merged?

no, spell-check is not fixed yet, how should we ignore the articles? all in the path (including the en)?

**Comment by tdelabro** [comment]
@d-roak What is spell check doing exactly? Why is it an issue?

**Comment by d-roak** [comment]
> @d-roak What is spell check doing exactly? Why is it an issue?

@tdelabro something we add it somewhere in time (before i arrived), that checks the english spelling in stuff... We can add files/paths to be ignored in the `.codespellignore`

**Pull request merged**